admin/views/tabs/metas/media.php

```diff
@@ -15,12 +15,8 @@
 	esc_html__( 'Media & attachment URLs', 'wordpress-seo' ),
 	dirname( __FILE__ ) . '/paper-content/media-content.php',
 	[
-		'help_text' => new WPSEO_Admin_Help_Panel(
-			'search-appearance-media',
-			__( 'Learn more about the Media and attachment URLs setting', 'wordpress-seo' ),
-			__( 'When you upload media (an image or video for example) to WordPress, it doesn\'t just save the media, it creates an attachment URL for it. These attachment pages are quite empty: they contain the media item and maybe a title if you entered one. Because of that, if you never use these attachment URLs, it\'s better to disable them, and redirect them to the media item itself.', 'wordpress-seo' ),
-			'has-wrapper'
-		),
+		// @todo fix the shortlink below to be a real yoa.st shortlink to the correct page!
+		'help_text' => new WPSEO_Admin_Help_Button( 'shortlink', __( 'Learn more about the Media and attachment URLs setting', 'wordpress-seo' ) ),
```

Comment:
Seems like you need to also update `paper-collapsible.php` to just echo `$help_text` instead of `$helptext->get_button_html`.

Reply:
Didn't change that bit, and yes I probably should :P